packages/react-native/src/primitives/Label/types.ts

```diff
@@ -1,8 +1,45 @@
 import { TextProps, TextStyle } from 'react-native';
 
-export interface LabelProps extends TextProps {}
+export interface LabelProps extends TextProps {
+  /**
+   * @description
+   * The variation property will affect the color of the Label.
+   */
+  variation?: LabelVariation;
```

Comment:
Just for context, I think the reason we used `variation` for React web is because we wanted to avoid confusion with Figma's "variants" (is that right, @dbanksdesign?). Perhaps this is something we should consider for RN as well